src/frontend-pwa/src/views/Report/Report.tsx

```diff
@@ -1,12 +1,131 @@
-import { TextField } from '@mui/material';
+/* eslint-disable react/no-array-index-key */
+/**
+ * TODO:    - implement error handling
+ *          - make less ugly
+ *
+ * @summary - Event reporting page for the Wayfinder application.
+ *          - Users can submit reports about events they witness.
+ *
+ * @author  TylerMaloney
+ */
+
+import React, { useState, useEffect } from 'react';
+import { Button } from '../../components/Button/Button';
+import { StyledReportContainer, StyledReportOuterDiv, StyledTextArea } from './report.styles';
+
+export function Report() {
+  const [eventType, setEventType] = useState('');
+  const [details, setDetails] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [latitude, setLatitude] = useState(0);
```

Comment:
Get Dallas to go over the localStorage stuff he set up to integrate that functionality into fallback states and error handling